### PR TITLE
Change "twine" format to "html", matching the Treaty. Fixes #24

### LINKED ISSUE #24: Babel returns format "twine" but that's not a valid format according to the treaty
According to <https://babel.ifarchive.org/babel_rev10.html#format>, there are 13 valid `<format>` values:

> `zcode, glulx, tads2, tads3, hugo, alan, adrift, level9, agt, magscrolls, advsys, html, executable`

But try running `babel` on a Twine file, e.g. `./babel -format test/twine/Test-Game.html`

Expected: `Format: html`
Actual: `Format: twine`, which isn't on this list.

### PR DESCRIPTION
**Pull request by dfabulich**
The first commit just fixes a bug, bringing Twine games into compliance with the treaty.

The second commit doesn't follow the Treaty of Babel (but I'm hoping to amend the Treaty of Babel in the course of merging this PR).

I'm proposing to define IFIDs in HTML like this:

```html
<meta property="ifiction:ifid" content="247603CE-784D-4404-90F5-5CE6356963F7">
```

Note that I'm using the `property` attribute, not `name`. `name` is standardized in HTML. `property` comes from RDFa; browsers ignore it. Open Graph uses `property`, and it seems pretty straightforward, so I'm doing the same. https://ogp.me/

### CODE REVIEW

**Comment by dfabulich** [comment]
I've got another PR ready to go, stacked on top of this one, to support IFIDs in HTML. https://github.com/dfabulich/babel-tool/pull/1

**Comment by erkyrath** [comment]
I've merged this into a branch (https://github.com/iftechfoundation/babel-tool/tree/twine-html) and updated it to handle more cases correctly:

    <meta prefix="ifiction: http://babel.ifarchive.org/protocol/iFiction/" property="ifiction:ifid" content="448E73DF-2D2F-47E7-A494-A46B40D4CFB4">

    <script>
    /* UUID://448E73DF-2D2F-47E7-A494-A46B40D4CFB5// */
    </script>

There's also now a test for the legacy case where no IFID is specified at all, and the tool must construct HTML-MD5.

**Comment by dfabulich** [comment]
There are now three places to discuss this PR:

1. Here
2. The intfiction thread https://intfiction.org/t/babel-for-html/53945
3. The google groups thread https://groups.google.com/g/babel-if/c/1haByrmRVKY

I've provided feedback on Zarf's branch in the intfiction thread. I'm going to aim to discuss this mostly on intfiction for now.

File: test/html/Test-Game-UUID.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Test Game</title>
+<script>
+/* UUID://448E73DF-2D2F-47E7-A494-A46B40D4CFB5// */
```
Comment (dfabulich) [inline test/html/Test-Game-UUID.html:6]: As a nitpick, IFWiki recommended an HTML comment, `<!-- UUID://XXXXXXXXXXXXXXXX// -->` not a `<script>` JS comment. But either one works.

**Comment by dfabulich** [comment]
I've filed https://github.com/iftechfoundation/babel-tool/pull/27 to incorporate feedback from the intfiction thread.

**Comment by dfabulich** [comment]
Aaand I've closed #27 as I'm convinced by Zarf's remarks on the intfiction thread. I now think that this PR (which has merged in Zarf's additional fixes) is ready to merge.

**Pull request merged**